gulpfile.js

```diff
@@ -73,7 +76,27 @@ gulp.task('karma-watch', ['build'], function() {
   server.start();
 });
 
+gulp.task('karma-sauce', ['build'], function() {
+  var customLaunchers = geSaLaKaCuLa({
```

Comment:
I would prefer to have this in `karma.conf.js` but reKala won't pick up on the launchers if they aren't passed here :|